src/basic/LayoutManager.cc

```diff
@@ -138,9 +138,9 @@ BasicSceneNode* MagMLLayoutManager::operator()(BasicSceneNode* parent, BasicPosi
 	 DisplayType display = layout.display();
 	 
 	 if ( actions_.empty() ) {
-		 	actions_[INLINE] = &MagMLLayoutManager::inline_display;
-		 	actions_[ABSOLUTE] = &MagMLLayoutManager::absolute;
-		 	actions_[BLOCK] = &MagMLLayoutManager::block;
+			actions_[M_DT_INLINE] = &MagMLLayoutManager::inline_display;
```

Comment:
I wsa wandering why you used this convention M_DT ? what is the DT standing for ?

Reply:
Ah , OK it is not for dtip ;).. Makes also sense now !